main.go

```diff
@@ -74,6 +75,10 @@ func main() {
 		addrs, passwords, aliases = loadRedisArgs(*redisAddr, *redisPassword, *redisAlias, *separator)
 	}
 
+	if (len(addrs) == 0 || addrsOnlyContainLocalhost(addrs)) && isOnCloudFoundry() {
```

Comment:
I think it would make more sense to check CF settings when e.g. there's a flag `use-cf` present. I'm not familiar with Cloud Foundry, what do you think?

Reply:
I need to doublecheck how to set this flag for a CF deployment. If I find out how to, I will make this dependent on the flag.